tempora/schedule.py

```diff
@@ -118,7 +128,7 @@ class PeriodicCommandFixedDelay(PeriodicCommand):
     """
 
     @classmethod
-    def at_time(cls, at, delay, target):
+    def at_time(cls, at, delay, target) -> Self:  # type: ignore[override] # FIXME: inverted attribute order
```

Comment:
I don't see the inverted attribute order. I'll file an issue instead of adding a fixme comment.
```suggestion
    def at_time(cls, at, delay, target) -> Self:  # type: ignore[override]
```

Reply:
I've added an issue link to the comment. This way if the decision is to "leave it as is", it'll be documented and linked to